webpack.config.js

```diff
@@ -165,10 +158,10 @@ const webpackConfig = {
 	},
 	plugins: [
 		new ForkTsCheckerWebpackPlugin(),
-		new FixStyleOnlyEntriesPlugin(),
+		new RemoveEmptyScriptsPlugin(),
 		new CustomTemplatedPathPlugin( {
 			modulename( outputPath, data ) {
-				const entryName = get( data, [ 'chunk', 'name' ] );
+				const entryName = data && data.chunk && data.chunk.name;
```

Comment:
We don't actually have a direct dependency on lodash in our deps right now, so I figured, lets just remove it, its really not needed.